transformations/replace_financial_amounts/transformation.py

```diff
@@ -0,0 +1,437 @@
+from typing import List, Dict, Tuple
+import random
+import spacy
+import ftfy
+
+from transformations.replace_financial_amounts import entity_financial_amount
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+# Token has three keys => text: str, category: str and is_financial_amount: bool
+Token = Dict[str, any]
+
+
+def create_tokens(text: str) -> List[Token]:
+    """
+    Create a list of tokens that represents the input text.
+
+    Parameters:
+        `text`: The text or sentence to split into tokens.
+    """
```

Comment:
Please add what the function returns as well under "Returns" wherever it is missing.

Reply:
changed where applicable